index.bs

```diff
@@ -5384,8 +5384,9 @@ type.
 
 <pre class="grammar" id="prod-SingleType">
     SingleType :
-        NonAnyType
+        DistinguishableType
         "any"
+        PromiseType ε
```

Comment:
Do we need the "ε" bit?

Reply:
I suppose not (but could be proved wrong by someone whose grammar knowledge is more recent than mine)

Reply:
I found the ε confusing when working with the grammar. Was the original intention just to emphasize that Null deliberately isn’t included in this alternative?